src/sentry/workflow_engine/processors/workflow.py

```diff
@@ -414,7 +415,9 @@ def process_workflows(
             organization,
         ):
             for action in actions:
-                action.trigger(event_data, detector)
+                trigger_action.delay(
```

Comment:
thoughts on moving the metrics below into the `trigger_action.delay` method? that way the logging is co-located with the call (i've been meaning to move these into action.trigger 😅)